README.md

```diff
@@ -14,37 +14,64 @@ Information for package authors can be found at
 
 It's recommended to use `stack`: <http://docs.haskellstack.org>.
 
+### Build
+
 To build in development mode:
 
 ```
 $ stack build
 ```
 
-To run the server:
+To build in production mode:
 
 ```
-$ stack exec pursuit
+$ stack build --flag pursuit:-dev
 ```
 
-If you have content in your data directory
-(see [Database structure](#database-structure)) then the database will
-be regenerated before the server starts listening - this can take a short time
-depending on how much data you have.  The site should then be available at
-`http://localhost:3000`.
+### Develop
 
-To build in production mode:
+To iterate quickly during development, you can use `ghci`:
 
 ```
-$ stack build --flag pursuit:-dev
+$ stack ghci
 ```
 
-## Database structure
+Once the REPL has loaded, you can reload the code and then update the web server:
+
+```
+> :l DevelMain
+> update
+```
+
+### Web server
+
+To run the web server on <http://localhost:3000>:
+
+```
+$ stack exec pursuit
+```
+
+Make sure you have content in your `data/verified` directory (see
```

Comment:
Can we please remove this reference to "the `data/whatever` directory"? Since `./data` is just the default value of the configuration variable for the data directory this isn't necessarily true.